tests/test_config.py

```diff
@@ -91,3 +93,22 @@ def test_can_read_toml_rich_types(tmp_path):
 """
     )
     check_configuration_file_reader(config_file)
+
+
+def test_can_read_env_variable(tmp_path):
+    # Check that we can read the "regular" INI .pylintrc file
+    # if it has an environment variable.
+    os.environ["tmp_path_env"] = str(tmp_path)
+    config_file = (
+        Path(os.path.expandvars(os.path.expanduser("${tmp_path_env}"))) / ".pylintrc"
+    )
+
+    config_file.write_text(
```

Comment:
Shouldn't we call the function ``read_config_file`` in order to test what we want to test ?